independent-publisher-connectors/XSOAR/apiDefinition.swagger.json

```diff
@@ -0,0 +1,100 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Provides the ability to submit content to XSOAR integrations when using instance.execute.external to expose integration instances over HTTPS. Tested to work with the Generic Webhook integration.",
+    "version": "1.0.0",
+    "title": "XSOAR",
+    "contact": {
+      "name": "Landon Chelf",
+      "email": "landon.chelf@fisglobal.com"
+      "url": "https://fisglobal.com"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "demisto.live",
+  "basePath": "/instance/execute",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.paloaltonetworks.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.paloaltonetworks.com/legal-notices/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "IT Operations"
+    }
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/": {
+      "post": {
+        "summary": "Send to XSOAR",
+        "description": "Send JSON data to XSOAR.",
+        "operationId": "SendToXSOAR",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "request_body",
+            "description": "Valid JSON content",
+            "required": true,
+            "schema": {
+              "type": "object"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "401": {
+            "description": "API key is missing or invalid",
```

Comment:
@landonc No response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**